docs/integration-guide.md

```diff
@@ -56,15 +56,15 @@ In ACME, it's possible to create one account and use it for all authorizations a
 
 However, for most larger hosting providers we recommend using a single account and guarding the corresponding account key well. This makes it easier to identify certificates belonging to the same entity, easier to keep contact information up-to-date, and easier to provide rate limits adjustments if needed. We will be unable to effectively adjust rate limits if many different accounts are used.
 
-# Few Names Per Certificate or Many?
+# Multi-domain (SAN) Certificates
 
-Our [rate limits](/docs/rate-limits/) allow for up to 100 names per certificate. How to distribute names across certificates is largely a question of individual needs.
+Our [issuance policy](/docs/rate-limits/) allows for up to 100 names per certificate. Whether you use a separate certificate for every hostname, or group together many hostnames on a small number of certificates, is up to you.
 
-Favouring many certificates with few names on each means less moving parts are required to logically add and remove domains as they are provisioned and retired. Less names per certificate also means a smaller overall certificate size.
+Separate certificates per hostname on each means fewer moving parts are required to logically add and remove domains as they are provisioned and retired. Separate certificates also minimize certificate size, which can speed up HTTPS handshakes on low-bandwidth networks.
```

Comment:
> Separate certificates also minimize certificate size, which can speed up HTTPS handshakes on low-bandwidth networks.

I think this should be "minimizes". 

I had intentionally left out any claims to performance because I'm fairly certain that <100 ~average size SAN entries would mean that the certificate always fits inside the same number of TLS records and the performance win might not be that large. I don't know that anyone will read it that nit-picky but I thought it was best to keep that claim vagueish because it needs further investigation to be certain.